### ISSUE TITLE
Allow Reordering of User Lists

### ISSUE THREAD

**Issue by Nayelianne** [bluesky-social/social-app#5107]
**Is your feature request related to a problem? Please describe.**

Currently User Lists are ordered by creation date, with the most recent ones at the top, which can be inconvenient when a user has many lists and the most used ones end up all the way at the bottom.

**Describe the solution you'd like**

Ideally, allow the user to manually reorder their own lists through dragging, according to their own preferences.
Alternatively, but possibly more work: Dynamic filters to sort by "Alphabetical" / "Recently Updated" / "Creation Date" for ordering them.

**Comment by adamatic-me** [comment]
I was about to make this same request. Although to make it easy for devs, I was just going to ask for changing it to alphabetical for now. Alphabetical would be useful for users browsing another user list, where as the creation date sort is pretty useless from that pov.

**Comment by quinncomendant** [comment]
I agree – alphabetically would be significantly better than the current sort order, until something more complex is built. Users would be able to set the order of lists by changing their name. Just need a sort function added to [MyLists.tsx](https://github.com/bluesky-social/social-app/blob/main/src/view/com/lists/MyLists.tsx).

**Comment by ghost** [comment]
Yes! It's been pretty frustrating adding accounts to lists and having to remember where it is instead of ordering the lists with my most used lists at the top.

Additionally, it would be great if I could first to choose to add the account to a moderation list or a content list. There's no reason for me to see block lists if I'm trying to add someone to a content list, and vice versa. In fact, mixing up the mod lists and the content lists means I'm likely to accidentally add someone whose posts I want to see to a mute/block lists.

**Comment by o-simardcasanova-net** [comment]
Seconding this, being able to manually reorder lists and to reorder them based on different criteria (alphabetical, etc.) would be terrific for power users.
